doc/sphinx/source/recipes/recipe_ipccwg1ar6ch3.rst

```diff
@@ -35,13 +37,21 @@ following figures from `Eyring et al. (2021)`_ can currently be reproduced:
 .. _`Eyring et al., 2021`: https://www.ipcc.ch/report/ar6/wg1/chapter/chapter-3/
 .. _`Eyring et al. (2021)`: https://www.ipcc.ch/report/ar6/wg1/chapter/chapter-3/
 
+To reproduce Fig. 3.9 please upload the shapefile of the [AR6 reference regions]
+(https://github.com/SantanderMetGroup/ATLAS/tree/v1.6/reference-regions)
+([IPCC-WGI-reference-regions-v4_shapefile.zip]
+(https://github.com/SantanderMetGroup/ATLAS/blob/v1.6/reference-regions/IPCC-WGI-reference-regions-v4_shapefile.zip))
+and store it in `<auxiliary_data_dir>/IPCC-regions/` (the `auxiliary_data_dir`
+is defined in the `config-user.yml` file).
```

Comment:
I see both, the text and the linked URL including the parentheses when I preview the .rst file. Maybe double-check if the syntax is correct.